src/components/box/Box.tsx

```diff
@@ -65,6 +65,7 @@ export type BoxProps = Partial<
     paddingTop: Padding;
     style: CSSProperties;
     textAlign: 'center' | 'left' | 'right';
+    [key: string]: unknown;
```

Comment:
its safer to explicitly set props, so I would say use `to: string`

Reply:
The problem with that is the `element` prop can be used for both html elements and React components.
So if you have a component `Foo` that accepts a prop `bar: string`
and you do `<Box element={Foo} />` it's not possible to add the required `bar` prop.